src/api/EntityDataModelApi.js

```diff
@@ -1726,6 +1726,8 @@ export function getAssociationTypeDetails(associationTypeId :UUID) :Promise<*> {
 /**
  * `GET /edm/association/type/{uuid}/available`
  *
+ * Gets all available assocations for the given EntityType UUID.
```

Comment:
Please change `associations` to `AssociationTypes`.

Reply:
Done.